spec/unit/knife/core/bootstrap_context_spec.rb

```diff
@@ -20,6 +20,10 @@
 require "chef/knife/core/bootstrap_context"
 
 describe Chef::Knife::Core::BootstrapContext do
+  before do
+    Chef::Config[:fips] = false
```

Comment:
That makes sense. Probably worth adding a comment about that in the tests since it behaves differently (or at least appears to) than other config settings.